src/data/monoid_algebra.lean

```diff
@@ -325,8 +329,57 @@ lemma alg_hom_ext ⦃φ₁ φ₂ : monoid_algebra k G →ₐ[k] R⦄
 
 end lift
 
--- TODO we should prove here that G and k commute;
--- presumably a `linear_mul_action` typeclass is in order
+section
+
+variables (k)
+/-- When `V` is a `k[G]`-module, multiplication by a group element `g` is a `k`-linear map. -/
+-- FIXME: this takes incredibly long to elaborate, and I don't know why.
```

Comment:
Should we investigate this before merging?

Reply:
Yes, I would love some help with identifying the problem here.

I *think* it's typeclass resolution going off the rails, but I'm still not that great at diagnosing these.

Reply:
It seems this is back under the official new limit of `-T100000`, so I'm going to stick a fork in it and declare it okay. :-)